components/PageError.tsx

```diff
@@ -5,7 +5,7 @@ import { Terminal } from '@components/PageRenderError';
 
 export const PageError = ({ title, error }: { title: string; error: string }): JSX.Element => {
     return (
-        <main className="container mx-auto px-16 flex-grow">
+        <main className="container flex flex-col flex-grow mx-auto px-16 overflow-auto">
```

Comment:
What's the `overflow-auto` does here?

Reply:
It prevents content from exceeding the container when the error is very long.

Reply:
It prevents content from exceeding the container when the error is very long.